pycheribuild/config/target_info.py

```diff
@@ -491,7 +490,7 @@ def cheri_config_suffix(self, config: "CheriConfig"):
             # MIPS supports 128/256 -> include that in the configuration
             result += config.mips_cheri_bits_str
         if self.is_hybrid_or_purecap_cheri():
-            if config.cheri_cap_table_abi != self.DEFAULT_CAP_TABLE_ABI:
```

Comment:
I guess it would be good to not use a separate if it matches the default but since this is an option for very few people I think "don't pass the wrong argument" is a reasonable solution.